readme.md

```diff
@@ -55,9 +64,15 @@ It just so happens you will be learning one of the best ORM's on the market. It
 
 > Active Record is the M in MVC - the model - which is the layer of the system responsible for representing business data and logic. Active Record facilitates the creation and use of business objects whose data requires persistent storage to a database. It is an implementation of the Active Record pattern which itself is a description of an Object Relational Mapping system. - Taken from AR docs
 
-## Active Record
+## ActiveRecord
```

Comment:
Referred to as "Active Record" in the official documentation: https://github.com/rails/rails/tree/master/activerecord